content/docs/command-reference/add.md

```diff
@@ -58,6 +58,8 @@ files that can be easily tracked with Git.
 
 It's possible to prevent files or directories from being added by DVC by adding
 the corresponding patterns in a [`.dvcignore`](/doc/user-guide/dvcignore) file.
+You can also undo the action of adding files or directories using `dvc add` by
+following this [guide](/docs/user-guide/how-to/undo-adding-data).
```

Comment:
> undo the action of adding files or directories using `dvc add`

That sounds really redundant, especially given the context. Can we simplify the phrase? Simple, direct language please 🙂.

Also, it's better to link part of the natural sentence instead of adding something like: "by following this [link]()" when possible.

Reply:
Also, this sentence isn't that closely related to the paragraph you put it in. It may be the right place for this info, but let's make it a separate paragraph?